datagovharvester/validate/dcat_us.py

```diff
@@ -0,0 +1,31 @@
+from jsonschema.exceptions import ValidationError
+from jsonschema import Draft202012Validator
+
+
+def parse_errors(errors):
+    error_message = ""
+
+    for error in errors:
+        error_message += (
+            f"error: {error.message}. offending element: {error.json_path} \n"
+        )
+
+    return error_message
```

Comment:
Is this the best consolidation of errors that makes sense?  A single string of an unknown number of errors?  Isn't a list of strings more flexible?